### PR TITLE
chore: refactor container-styles

### PR DESCRIPTION
**Pull request by luqven**
This PR simplifies the styling ternary logic of the `AssetCard.tsx` component.

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #235** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/235" target="_blank">Graphite</a> 
    * **PR #236** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/236" target="_blank">Graphite</a> 
      * **PR #237** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/237" target="_blank">Graphite</a> 
        * **PR #238** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/238" target="_blank">Graphite</a> 
          * **PR #239** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/239" target="_blank">Graphite</a> 
            * **PR #240** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/240" target="_blank">Graphite</a> 
              * **PR #241** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/241" target="_blank">Graphite</a> 
                * **PR #242** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/242" target="_blank">Graphite</a> 
                  * **PR #243** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/243" target="_blank">Graphite</a> 
                    * **PR #244** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/244" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: frontend/src/components/card/AssetCard.tsx [hunk 1]
```diff
@@ -34,8 +34,8 @@ export function AssetCard({
   const containerStyles = [
     styles.container,
     layout === "list" ? styles.list : styles.grid,
-    !isImage ? styles.disabled : "",
-    selectedAssetId === asset.id ? styles.selected : "",
+    !isImage && styles.disabled,
+    selectedAssetId === asset.id && styles.selected,
```
Comment (frederickfogerty) [inline frontend/src/components/card/AssetCard.tsx:38]: Does this not add `false` to the class list?

**Review by frederickfogerty — CHANGES_REQUESTED** [review]
See inline comments

File: frontend/src/components/card/AssetCard.tsx (hunk 1 above)
Comment (luqven) [reply frontend/src/components/card/AssetCard.tsx:38]: 💯 , I'll refactor this so this doesn't set the false value as the class name

**Pull request merged**